robottelo/cli/host.py

```diff
@@ -126,10 +126,13 @@ def facts(cls, options=None):
         return facts
 
     @classmethod
-    def package_install(cls, options):
+    def package_install(cls, hosts='', package=''):
         """Install packages remotely."""
-        cls.command_sub = 'package install'
-        return cls.execute(cls._construct_command(options), output_format='csv')
+
+        return cls.execute(
+            f'job-invocation create --feature katello_package_install'
+            f' --search-query "{hosts}" --inputs "package={package}"'
+        )
```

Comment:
@lhellebr Issue linked is expected after 6.10, but `Host` CLI's still uses katello-agent, and it just exists to keep warning users to use rex instead of katello-agent, so it doesn't make sense to update Host CLI's for achieving this.
Please consider updating the test instead to use `JobInvocation` CLI options to package_install, check this test for reference  https://github.com/SatelliteQE/robottelo/blob/master/tests/foreman/cli/test_host.py#L1912

Reply:
That file implements hammer host commands, so you would not change that unless the command was removed  or changed completely. You could update the description if you like.

`~]# hammer host package install --host-id 1 --packages vim
This command uses katello agent and will be removed in favor of remote execution in a future release.
The remote execution equivalent is `hammer job-invocation create --feature katello_package_install`.`

Reply:
Ping @lhellebr ?

Reply:
@jyejare Pong, I continued the discussion in followup comments, is there something specific in this thread you need a reply to?

Reply:
@lhellebr Please reverses the change to the `def package_install` function. That function implements a hammer command that still exists (someone else might need it).  There are other ways of installing packages depending on where you just need the package to be there, e.g. a setup step (search for run and execute examples which use SSH), or if you want to use Satellite to install it as part of the test (see https://github.com/SatelliteQE/robottelo/pull/9220/files#r763746302)

Reply:
@lhellebr Ahh I thought the conversation in unresolved, so i just pinged you here!

Reply:
@swadeley `someone else might need it` - the function fails by default though! However, I understand it's supposed to implement a certain hammer command, that's why I'm still not sure what is the cleanest solution here.